xarray/core/rolling_exp.py

```diff
@@ -1,4 +1,5 @@
 import numpy as np
+from pandas.core.common import count_not_none
```

Comment:
Can you add a TODO note about vendoring this, too? I don't think this is a public pandas API.

(CC @max-sixty )

Reply:
Copy-pasted the whole thing in xarray.core.pdcompat

Reply:
great, thanks!